src/Parser/Core/Modules/StatTracker.js

```diff
@@ -0,0 +1,257 @@
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import SPECS from 'common/SPECS';
+import { calculateSecondaryStatDefault } from 'common/stats';
+import { formatMilliseconds } from 'common/format';
+
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+const debug = false;
+
+// TODO: add stam, speed, avoidance, leech
```

Comment:
I think adding them right away would be for the best.

Reply:
yeah I'm going to add them before this gets merged, just pushing something out to be reviewed